README.md

```diff
@@ -39,17 +63,16 @@ an Islandora Foundation [Contributor License Agreement][6] or
 * American Philosophical Society
 * common media inc.
 
-## Maintainers
+## Development
 
-* [Daniel Lamb](https://github.com/dannylamb)
+If you would like to contribute, please get involved by attending our weekly [Tech Call](https://github.com/Islandora-CLAW/CLAW/wiki). We love to hear from you!
```

Comment:
https://github.com/Islandora/documentation/wiki

Reply:
this should be fixed now